action.yml

```diff
@@ -40,3 +40,11 @@ inputs:
     description: 'Close stale discussions as answered'
     required: false
     default: false
+  github-bot:
+    description: 'Github action bot login name'
+    required: false
+    default: 'github-actions'
+  page_size:
```

Comment:
> I'm not sure why anyone would need to configure page size, could you provide some context?
> 
> Also if we're going to keep this, let's define it as `page-size` to keep it consistent.

This page_size=50 is defined for our repo since we have too many discussions. If anyone is using it, might need to make the count higher or lower, so I intentionally moved it to action.yml.